cloudtasks/api/TasksSample/CreateTask.cs

```diff
@@ -50,5 +51,11 @@ class CreateAppEngineTask
         Console.WriteLine($"Created Task {response.Name}");
         return response.Name;
     }
+
+    public static void Main(string[] args)
+    {
+        var snippet = new CreateAppEngineTask();
+        snippet.CreateTask();
+    }
```

Comment:
We have been streadily moving away from Main methods in C#, we don't need them as we can have parameters with default values.
Any particular reason this is needed here?

Reply:
Clarified internally - this has since been reverted.